.github/workflows/process-results.yml

```
@@ -0,0 +1,32 @@
+name: Process benchmark results
+
+on: push
+
+jobs:
+  run-benchmark-monitor:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout processing script
+      uses: actions/checkout@v3
+      with:
+        repository: cz4rs/benchmark_monitor
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        path: benchmark-results
+
+    - name: Process available benchmark results
+      run: |-
+        python3 -m venv env
+        source env/bin/activate
+        pip install wheel
+        pip install -r requirements.txt
+        python benchmark_monitor.py -d ./benchmark-results -o output
+
+    - name: Deploy
+      uses: JamesIves/github-pages-deploy-action@v4
+      with:
+        folder: output
+        repository-name: cz4rs/cz4rs.github.io
```

FIXME:
```suggestion
        repository-name: kokkos/kokkos-core-wiki
```
(work out the details for deployment)

---

Use this (`kokkos-benchmark-results`) repository instead.